modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -202,11 +215,44 @@ The Server Key is required to uniquely identify the server configuration. The re
 === Anypoint MQ Config
 
 [cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
-|======================
+|===
 | Field | Type | Description | Default Value | Required
 | Server Key | String | Identifier for the server configuration in AsyncAPI | None | Yes
 | Config Ref | String | Reference to the Anypoint MQ configuration details | None | No
-|======================
+|===
+
+[NOTE]
+The Server Key is required to uniquely identify the server configuration. Config Ref is optional and used to specify additional configuration details.
+
+[[SolaceConfig]]
+=== Solace Config
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|===
+| Field | Type | Description | Default Value | Required
+| Server Key | String | Server key set in the AsyncAPI spec | None | Yes
+| Config Ref | String | Reference to the configuration name of the Solace connection | None | No
+|===
+
+[NOTE]
+The Server Key is required to uniquely identify the server configuration. Config Ref is optional and used to specify additional configuration details.
+
+=== Limitations of Solace Connector
+
+The Solace connector in APIkit for AsyncAPI has the following limitations:
+
+* The Solace Connector supports `queue` and `topic` for `destinations` in the `message listener` and `publish` operations. But the APIkit for AsyncAPI implementation only supports `queue` as `destination`.
```

Comment:
```suggestion
* Solace Connector supports `queue` and `topic` for `destinations` in the `message listener` and `publish` operations. But the APIkit for AsyncAPI implementation only supports `queue` as `destination`.
```

Reply:
Connecter operations are bold...?

Reply:
We try to avoid using UI (or other) elements as parts of speech.

"supports using `queue` as the value of `destination`."

Reply:
OK! Thanks!